scripts/start.py

```diff
@@ -58,11 +54,10 @@
     action='store_true')
 _PARSER.add_argument(
     '--disable_host_checking',
-    help=(
-        'optional; if specified, disables host checking so that the dev '
-        'server can be accessed by any device on the same network using the '
-        'host device\'s IP address. DO NOT use this flag if you\'re running '
-        'on an untrusted network.'),
+    help='optional; if specified, disables host checking so that the dev '
```

Comment:
Please leave this as it was before; ditto below.

Reply:
The new is actually more commonly used in the other files, and looks much nicer IMO, so I'd prefer we keep it like this. WDYT?

https://github.com/oppia/oppia/blob/35eaf75fb4243fbd957f5e937b62c3ebea75cf07/scripts/run_e2e_tests.py#L111-L116
https://github.com/oppia/oppia/blob/35eaf75fb4243fbd957f5e937b62c3ebea75cf07/scripts/run_e2e_tests.py#L119-L129
https://github.com/oppia/oppia/blob/35eaf75fb4243fbd957f5e937b62c3ebea75cf07/scripts/run_e2e_tests.py#L135-L156

Reply:
I think I'd prefer a single convention throughout for indentation -- i.e. only do it when there's a paren at the end of the previous line, and always indent by exactly 4.

I won't insist, though.